spdx/writers/json.py

```diff
@@ -14,6 +14,20 @@
 from spdx.writers.tagvalue import InvalidDocumentError
 from spdx.writers.jsonyamlxml import Writer
 from spdx.parsers.loggers import ErrorMessages
+import numpy as np
```

Comment:
Can you remove the dependency on `numpy`?  You should be able to test for regular `int`, `float`, etc.